evaluation/config.json

```
@@ -0,0 +1,28 @@
+{
+    "testdata_path": "input/qa.jsonl",
+    "results_dir": "results/experiment<TIMESTAMP>",
```

As mentioned before, no need to have placeholders here. I would most probably do sth like the following generate nested directories like `experiment-{timestamp}`:
```suggestion
    "results_dir": "results",
```